src/controller.rs

```diff
@@ -253,6 +261,14 @@ impl TnuaController {
         Some((&boxable_action.input, &boxable_action.state))
     }
 
+    /// The currently running action, together with its state, as mutable.
+    /// Useful if you need to touch the state of a running action to respond to game events.
```

Comment:
I think maybe we should add a warning here that if the action is replaced the changes to the state are lost.

(same is true for the basis, but the basis is much less likely to change)